src/raftstore/store/worker/metrics.rs

```diff
@@ -74,4 +74,19 @@ lazy_static! {
             "tikv_raftstore_hash_duration_seconds",
             "Bucketed histogram of raftstore hash compution duration"
         ).unwrap();
+
+    pub static ref SNAPSHOT_KV_COUNT_HISTOGRAM: Histogram =
+        register_histogram!(
+            "tikv_snapshot_kv_count",
+            "Total number of kv in snapshot",
+             vec![1000.0, 10000.0, 100000.0, 1000000.0, 10000000.0]
+
+        ).unwrap();
+    pub static ref SNAPSHOT_SIZE_HISTOGRAM: Histogram =
+        register_histogram!(
+            "tikv_snapshot_size",
+            "Size of snapshot",
+             vec![1000.0, 10000.0, 100000.0, 1000000.0, 10000000.0,
```

Comment:
use `exponential_buckets(1024, 2.0, 22).unwrap()`, so we can cover from 1KB to 4GB.